build-reanimator-lttng.sh

```diff
@@ -209,7 +209,7 @@ runcmd cd "${repositoryDir}"
 runcmd cd reanimator-library
 runcmd chmod +x buildall.sh
 if [[ "${install}" == true ]]; then
```

Comment:
Instead of [[ "${install}" == true ]] you can just write:

if $install; then

because true and false are commands.